BottomWidgets/BottomWidget.h

```diff
@@ -92,6 +92,8 @@ private slots:
 
     bool AdjustingPos;
     quint64 posAdjust;
+
+    int sliderSongOriginalPos; //sliderSong被按下时的位置
```

Comment:
针对后面提到的精度不够的问题，我在本地修改了一下记录的对象，待会提交
```suggestion
    int audioOriginalPos; //音频被按下时的位置（为了记录下精确地时间，记录音频位置，之前记录slider位置精度不够会导致偏移）
```